tests/web/test_query_size.py

```diff
@@ -1,16 +1,19 @@
 import pytest
 
-from snuba.web.query import MAX_QUERY_SIZE_BYTES, get_query_size_group
+from snuba import settings
+from snuba.pipeline.stages.query_execution import get_query_size_group
 
-TENTH_PLUS_ONE = int(MAX_QUERY_SIZE_BYTES / 10) + 1
+TENTH_PLUS_ONE = int(settings.MAX_QUERY_SIZE_BYTES / 10) + 1
```

Comment:
the same `MAX_QUERY_SIZE_BYTES` is also in settings? I think you did this because you thought settings is a better place for it and I would agree. Why didnt you delete the one from query.py?

Reply:
good point